### ISSUE TITLE
Exception while using on a remote DB with approx 5.000 tables and views...

### ISSUE THREAD

**Issue by hidegh** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#357]
Error
Running transformation: System.Runtime.Remoting.RemotingException: Object '/01e5c879_5905_4413_9fe8_484c17c98cf3/7ok7i7ezvpur5qygvtjj+2k7_9.rem' has been disconnected or does not exist at the server.

Weird is, that the context get's generated (table white-list is around 20 tables). But when all the table and mappings are created inside the single *.cs file, just before splitting it, the error above occurs.

**So temporarily I solved this the following:**

Introduced a new variable inside Settings and replaced all Cmd.CommandTimeout = 600; with Cmd.CommandTimeout = Settings.CommandTimeout; inside EF.Reverse.POCO.Core.ttinclude file.

```
public static class Settings
    {
        // Main settings
        public static int CommandTimeout = 0;
```

**Also worth to add 2x2 sections inside the EF.Reverse.POCO.ttinclude file (remarks)** - custom pre-map code, custom post-map code and same section for the POCO class itself

**Comment by sjh37** [comment]
I've added this thanks, and will be in the next release.

**Comment by sjh37** [comment]
Released in v2.35.0
